src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/error/ErrorResponse.java

```diff
@@ -1,20 +1,20 @@
 package ca.bc.gov.open.jag.efilingapi.error;
 
 public enum ErrorResponse {
-    INVALIDROLE("INVROLE", "User does not have a valid role for this request."),
-    ACCOUNTEXCEPTION("MLTACCNT", "Client has multiple CSO profiles"),
-    GETPROFILESEXCEPTION("CSOERROR", "Calling CSO accountFacade.getProfiles caused an exception.");
+    INVALIDROLE( "User does not have a valid role for this request."),
+    ACCOUNTEXCEPTION("Client has multiple CSO profiles"),
```

Comment:
non-blocking: missing period if you use the other errors, or the others have a period. vice versa.

Reply:
line 7 as well

Reply:
fixed thanks!